mentoring/answer.py

```diff
@@ -127,6 +128,12 @@ def submit(self, submission):
             'score': 1 if self.status == 'correct' else 0,
         }
 
+    def submit(self, submission):
```

Comment:
@Kelketek nit: might make sense to move before `get_results` to reduce diff size.